### PR TITLE
pyocd in image for microbit

### PR DESCRIPTION
**Pull request by rienafairefr**
Fix a missing package, pyocd, to properly support microbit
Testesd working on devlille on a CI gateway

### CODE REVIEW

**Comment by aabadie** [comment]
I'm just wondering if the recipe should be located in the [python folder of the devtools](https://github.com/iot-lab/iot-lab-yocto/tree/master/meta-iotlab/recipes-devtools/python) instead, since it's using the setuptools.

File: meta-iotlab/recipes-devtools/pyocd/python-pyocd.bb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+SRC_URI[md5sum] = "3a7b1197fcc950d86a6d05ea83145764"
+SRC_URI[sha256sum] = "b89f41b1d6c1f81777220d0dfdaf42f73064709df9842cf81027812b4d7a1951"
+
+PV = '0.8.0'
+
+SRCNAME = "pyOCD"
+SRC_URI = "https://pypi.python.org/packages/35/c6/9fee7264155206498796066e924f6edef475b1b683f727e320219b1c85e2/pyOCD-0.8.0.tar.gz"
+
+S = "${WORKDIR}/${SRCNAME}-${PV}"
+inherit setuptools
```
Comment (aabadie) [inline meta-iotlab/recipes-devtools/pyocd/python-pyocd.bb]: There's a missing newline at the end of the file

**Comment by rienafairefr** [comment]
In the case of python-msp430-tools that was added for the tmotesky, also using setuptools, I've put it in recipes-devtools. I think you're right, this should all be in python/ subfolder

**Comment by rienafairefr** [comment]
I moved msp-430, cc2538, pyocd to the python/ folder, then fixed the dependencies in pyOCD (I needed to backport some bitbake recipes from meta-openembedded), tests are finally passing on devlille's microbit, with the image built on my laptop. Thanks @fsaintma for the help

File: meta-iotlab/recipes-devtools/python/python-future_0.16.0.bb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=3f457538b297bd0ef86564b8ac216ad7" 
+ 
+SRC_URI[md5sum] = "3e8e88a2bda48d54b1da7634d04760d7" 
+SRC_URI[sha256sum] = "e39ced1ab767b5936646cedba8bcce582398233d6a627067d4c6a454c90cfedb" 
+ 
+PYPI_PACKAGE_HASH = "99abde815842bc6e97d5a7806ad51236630da14ca2f3b1fce94c0bb94d3d" 
+ 
+inherit pypi setuptools 
+ 
+BBCLASSEXTEND = "native"
```
Comment (aabadie) [inline meta-iotlab/recipes-devtools/python/python-future_0.16.0.bb]: missing newline

File: meta-iotlab/recipes-devtools/python/python-websocket-client_1.0.bb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+LIC_FILES_CHKSUM = "file://LICENSE;md5=18b09a20dd186af4fd60f1a08311628c"
+
+inherit pypi setuptools
+SRCREV = "d97eadb070d685aac207fcf6d663a013db42d86a"
+
+# for now, prepopulate this in the downloads directory
+SRC_URI = "git://github.com/websocket-client/websocket-client.git"
+
+S = "${WORKDIR}/git"
+PYPI_PACKAGE = "websocket_client"
```
Comment (aabadie) [inline meta-iotlab/recipes-devtools/python/python-websocket-client_1.0.bb]: missing newline

File: meta-iotlab/recipes-devtools/python/python-pyocd_0.8.0.bb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+SRC_URI[md5sum] = "3a7b1197fcc950d86a6d05ea83145764"
+SRC_URI[sha256sum] = "b89f41b1d6c1f81777220d0dfdaf42f73064709df9842cf81027812b4d7a1951"
+
+S = "${WORKDIR}/pyOCD-${PV}"
+
+inherit setuptools
+
+RDEPENDS_${PN} += "python-pyusb python-websocket-client python-enum34 python-future python-backports-ssl"
+
+
```
Comment (aabadie) [inline meta-iotlab/recipes-devtools/python/python-pyocd_0.8.0.bb]: This empty line can be removed

**Pull request merged**